### PR TITLE
Fix emote tooltip template

### PR DESCRIPTION
**Pull request by Aaron128l**

### CODE REVIEW

File: src/modules/emotes/emote.js [hunk 1]
```diff
@@ -34,10 +34,10 @@ module.exports = class Emote {
         `;
 
         return `
-            <span class="balloon-wrapper bttv-emote ${providerClass} ${idClass}">
-                <img src="${html.escape(this.images['1x'])}" srcset="${srcset.join(', ')}" alt="${html.escape(this.code)}" class="chat-line__message--emote">
-                <div class="balloon balloon--tooltip balloon--up balloon--center mg-t-1" style="text-align: center;">${balloon}</div>
-            </span>
+            <div class="tw-tooltip-wrapper inline bttv-emote ${providerClass} ${idClass}">
+                <img src="${html.escape(this.images['1x'])}" srcset="${srcset.join(', ')}" alt="${html.escape(this.code)}" class="chat-line__message--emote ${providerClass} ${idClass}">
```
Comment (night) [inline src/modules/emotes/emote.js]: remove extra classes here...

**Comment by DayBr3ak** [comment]
I tried it and it works but the tooltip isn't styled, and christmas emotes are broken

**Comment by Aaron128l** [comment]
It works fine for all types of BetterTTV emotes, Twitch's tooltip isn't built for overlapping emotes. Also there is no styling added by Twitch.

**Comment by DayBr3ak** [comment]
Tooltip aside, what I mean is that christmas emotes don't overlap anymore because the css specify `span` but now emotes are `div`.

**Comment by Aaron128l** [comment]
I'm not sure what you mean, looks fine https://i.aaron128l.com/CdnnrXm.png

**Pull request merged**